pages/linux/pacman.md

```diff
@@ -25,3 +25,11 @@
 - list only the explicitly installed packages and versions
 
 `pacman -Qe`
+
+- find which package owns a certain file
+
+`pacman -Qo filename`
+
+- empty package cache to free up space
+
+`pacman -Scc`
```

Comment:
I'm happy to see any other useful commands or later changing these variations if needed :)